src/frontends/tensorflow/src/utils.hpp

```diff
@@ -83,122 +78,6 @@ void get_const_input(const NodeContext& node, int64_t input_index, std::vector<T
     FRONT_END_THROW("Node must be converted to Constant.");
 }
 
-// Taken from: tensorflow/core/grappler/optimizers/arithmetic_optimizer.cc
```

Comment:
We also don't need TF copyright above 
"* Copyright 2017 The TensorFlow Authors....
Please delete it